### PR TITLE
Add mkl to fedora

### PR DESCRIPTION
**Pull request by streeve**
No longer comes by default? Breaking in ECP-CoPA/Cabana#720

### CODE REVIEW

**Comment by streeve** [comment]
@junghans not sure what I'm doing wrong here, but this isn't installing MKL for intel as I'm intending

**Comment by dalg24** [comment]
@masterleinad please review

**Comment by streeve** [comment]
> isn't installing MKL for intel as I'm intending

Fixed with the strategy from kokkos/ci-containers

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -41,11 +41,17 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Convert build_args
+        id: build_args
+        run: |
+          echo "args<<EOF" >> $GITHUB_OUTPUT
+          echo ${{ matrix.config.build_args }} | sed 's/,/\n/g' >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```
Comment (junghans) [inline .github/workflows/continuous-integration-workflow.yml:49]: Thanks for stealing my code ;-)

**Review by junghans — APPROVED** [review]

**Comment by junghans** [comment]
> @masterleinad please review

This is not kokkos, so nothing for @masterleinad to do here.

**Comment by dalg24** [comment]
> > @masterleinad please review
> 
> This is not kokkos, so nothing for @masterleinad to do here.

My bad :)

**Pull request merged**